src/library/Uno.Material/Controls/Card.cs

```diff
@@ -13,24 +14,44 @@ public partial class Card : Control
 		private const string __openedState = "Opened";
 		private const string __closedState = "Closed";
 
-		#region Headear and SubHeader
-		public string Header
+		#region HeaderContent and HeaderContentTemplate
```

Comment:
With this many properties we can probably move them into a separate file with `partial class`?

Reply:
Yes good idea, I will adjust that :) thanks for the feedback @XiaotianNetlift !